piwik/README.md

```diff
@@ -14,55 +13,44 @@ To use this addon you need a [piwik](http://piwik.org/) installation.
 Where to find
 -------------
 
-In the Friendica addon git repository `/piwik/piwik.php` and a CSS file for
-styling the opt-out notice.
+In the Friendica addon git repository `/piwik/piwik.php` and a CSS file for styling the opt-out notice.
 
 Configuration
 -------------
 
-The easiest way to configure this addon is by activating the admin panels of
-your ~friendica server and then enter the needed details on the config page
-for the addon.
+The easiest way to configure this addon is by activating the admin panels of your ~friendica server and then enter the needed details on the config page for the addon.
 
-If you don't want to use the admin panel, you can configure the addon through
-the .htconfig file.
+If you don't want to use the admin panel, you can configure the addon through the config/local.ini.php file.
 
-Open the .htconfig.php file and add "piwik" to the list of activated addons.
+Open the config/local.ini.php file and add "piwik" to the list of activated addons.
 
-    $a->config['system']['addon'] = "piwik, ..."
+    [system]
+	addon = ...,piwik
 
-You have to add 4 more configuration variables for the addon:
+You can change 4 more configuration variables for the addon:
 
-    $a->config['piwik']['baseurl'] = 'example.com/piwik/';
-    $a->config['piwik']['sideid'] = '1';
-    $a->config['piwik']['optout'] = true;
-    $a->config['piwik']['async'] = false;
+    [piwik]
+	baseurl = example.com/piwik/
```

Comment:
Is the non indentation of the baseurl line intended?

Reply:
It's a space/tab indentation issue.